active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Active Learning for caltech256 using Cleanlab"
+   ]
+  },
+  {
```

Comment:
In the few sections of text I edited, I spotted many typos. Please go through all text and fix typos.

Also when mentioning variable names in the markdown text, you should use backticks to make sure the variable name is properly formatted in a code font.